website/content/docs/job-specification/periodic.mdx

```diff
@@ -59,6 +59,9 @@ consistent evaluation when Nomad spans multiple time zones.
   prevents this job from running on the `cron` schedule but prevents force
   launches.
 
+See the [parameterized] documentation for additional considerations when
+using in conjunction with parameterized options.
+
```

Comment:
this seems a little buried under these Parameters. maybe move it up under Requirements?

Reply:
Since this periodic page doesn't talk about parameterized options at all, what about moving this line right before the `# `periodic` Requirements` section?

Reply:
It is a bit of a corner case, I didn't want to put it in the general description